exla/Makefile

```diff
@@ -44,8 +44,8 @@ $(EXLA_SO): $(EXLA_CACHE_SO)
 		cp -a $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
 		cp -a $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
 	else \
-		ln -sf $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
-		ln -sf $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
+		ln -srf $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
+		ln -srf $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
```

Comment:
This option isn't available on Mac

Reply:
Unfortunately realpath is only available on the latest macOS Ventura. So my suggestion is to check if there  is a  `realpath` and act accordingly.